frontend/internal-packages/agent/package.json

```diff
@@ -55,6 +55,7 @@
     "lint:biome": "biome check .",
     "lint:eslint": "eslint .",
     "lint:tsc": "tsc --noEmit",
-    "test": "vitest --watch=false --passWithNoTests"
+    "test": "vitest --watch=false --passWithNoTests",
+    "test:integration": "vitest --watch=false --passWithNoTests --config vitest.config.integration.ts"
```

Comment:
LGTM! 

Q. The interleaved output from parallel test execution seems like a good tradeoff for performance. I assume this was intentional - correct?

Reply:
That's right 👍🏻 However, some documentation may be necessary, as it is not intuitive for AI agents about assuming stand-alone execution and the lack of assertion. We would like to improve this as we use it.